types/model.d.ts

```diff
@@ -0,0 +1,161 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: loopback-datasource-juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/// <reference types="node" />
+
+import {AnyObject, Options, Callback, PromiseOrVoid} from './common';
+import {DataSource} from './datasource';
+import {EventEmitter} from 'events';
+
+// tslint:disable:no-any
+// tslint:disable:max-line-length
```

Comment:
AFAICT, we don't have `tslint` integration in juggler, therefore we should not need any `tslint` overrides.